### DISCUSSION TITLE
Flipping keys but leaving thumbs

### DISCUSSION THREAD

**Discussion by abandon42** [manna-harbour/miryoku#159] [Q&A] [answered]
After struggling with the default nav keys for months, I've finally discovered the magic of INVERTEDT. 

Flip Layers Inverted-T Nav looks exactly like what I'm looking for, but I'd like to avoid breaking my thumbkey habits. 

Is this a bad idea somehow?

**Comment by manna-harbour** [comment, marked as answer]
The main problem is that by default the Base right hand thumbs match those in Nav. There may not be any particular issue in losing that other than the inconsistency, so if you'd like to try it you could build with flip layers and inverted-T Nav, and substitute a custom Base with https://github.com/manna-harbour/miryoku/discussions/85.

**Reply by abandon42** [reply]
Thanks, you're awesome as always!

Is there a way to generate the regular invertedt keymap images? 

I finally got my hands on a keyboardio model 100 and I'm a bit disappointed that it's not a platform you support.

**Reply by manna-harbour** [reply]
> Is there a way to generate the regular invertedt keymap images?

See https://github.com/manna-harbour/miryoku/discussions/71#discussioncomment-2942890 plus the rest of that Discussion plus the linked Issue.

> I finally got my hands on a keyboardio model 100 and I'm a bit disappointed that it's not a platform you support.

There's no Miryoku Kaleidoscope (yet, it's on the [list](https://github.com/manna-harbour/miryoku/issues/90)), but you can use Miryoku KMonad, or [𝑥MK](https://github.com/manna-harbour/xmk) with Miryoku QMK or Miryoku ZMK.

Also, if someone adds Model 100 support to [QMK](https://community.keyboard.io/t/model-100-support-in-qmk/5537/8) or ZMK, I can add support to Miryoku QMK or Miryoku ZMK.

**Reply by javjar** [reply]
Hi! Any updates on Kaleidoscope support?
